### ISSUE TITLE
Feature: automatically bump Cargo.toml version

### ISSUE THREAD

**Issue by dbkaplun** [rust-lang-nursery/rust-semverver#61]
Hello,

It would be great if `cargo semver` had a subcommand that could automate a chunk of the publishing process and update the version in Cargo.toml. This would make make deployment less error-prone by reducing human interaction.

Thanks for considering!

**Comment by jrmuizel** [comment]
The 'cargo release' crate does a bunch of the manual work around publishing
a release. It has an open issue for integrating with rust-semverver.

**Comment by ibabushkin** [comment]
I agree -- maybe this functionality shouldn't be implemented from scratch in rust-semverver. I'll reach out to the cargo-release authors and see how this goes.
